acceptance/bundle/templates/default-python/integration_classic/script

```diff
@@ -3,6 +3,11 @@ venv_activate
 trace python -c 'import sys; print("%s.%s" % sys.version_info[:2])'
 uv pip install -q setuptools
 
+# Note(@pietern): without installing hatchling directly,
+# this test fails with Python 3.13 on Windows.
+# I cannot reproduce with `-keeptmp` and then manually running `uv build --wheel`...
```

Comment:
Is the comment re keeptmp still relevant after long paths are enabled?

Reply:
Thanks for flagging. It no longer reproduces with long paths enabled. That was the root cause...